watchdog/scripts/deploy.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
```

Comment:
I don't think this script would be useful really in production. I expect that we'll assign controllership of the watchdog to the NNS and it would then be only upgradable through proposals.

Reply:
Done, removed.